sqlmesh/core/engine_adapter/__init__.py

```diff
@@ -27,13 +27,16 @@
     "mssql": EngineAdapterWithIndexSupport,
 }
 
+DIALECT_ALIASES = {
+    "postgresql": "postgres",
```

Comment:
what's the point of dialect aliases? just expectp eople to put in one

Reply:
@izeigerman I believe you originally defined this alias (I just moved it to a dict). Do you have context on why this was done?